lib/utils/api.js

```diff
@@ -69,4 +69,29 @@ export async function getUserByType({ id, type }) {
   }
 }
 
+export async function registerMentor({
+  user_id,
+  first_name,
+  last_name,
+  mobile,
+  photo,
+  // birthday,
+  major,
+}) {
+  const config = {
```

Comment:
I think if we remove the global headers' definition it will also add `application/json` to the requests which means we don't need to specify in this request too.

Reply:
This should be one of two request where the payload is not JSON, so let's leave it like it is for now.